### PR TITLE
Previously watched

### PR DESCRIPTION
**Pull request by nathanielmillard**
## What's this PR do?
-Renames the GenrePage to Movie List
-Recycles this components to create a previously watched page

### How should this be manually tested?
-Navigate to previously watched from the nav menu

#### Screenshots (if appropriate)
Screen Shot 2021-02-09 at 2 46 48 PM

### CODE REVIEW

File: src/containers/MovieList/MovieList.scss [hunk 1]
```diff
@@ -3,6 +3,11 @@
   scroll-padding-top: 0;
 }
 
+.previously-watched {
+  background-image: linear-gradient(to right, #FDE997, #00D8FF);
+  scroll-padding-top: 0;
+}
+
```
Comment (sschipke) [inline src/containers/MovieList/MovieList.scss:10]: Per the comp, I believe this background gradient should be #FDE997 to #FF3399
[Hex Values Here](https://docs.google.com/document/d/18rHWoStJw2ZglnNJvR-GAYEimz3G4wb1BwgdejUGQ98)
## Screenshot of comp
Screen Shot 2021-02-09 at 4 03 48 PM

**Review by sschipke — CHANGES_REQUESTED** [review]
### Observations:

- [x] When I look at a specific movie from the "Previously Watched List" the runtime is displaying as `NAN`
Screen Shot 2021-02-09 at 4 12 37 PM

- [x] In the mobile view, the previously watched movies are rendered outside of their container and off the page 
602305b3cfe20e00078e5fca--blackstories-dev netlify app_previouslywatched(iPhone 6_7_8)

- [x] The film information on the "Previously Watched Page" should display only the following (see comp for examples): 
- Movie Poster
- Title
- Date Watched (`date_watched`)
- Who in the group chose the movie (`chosen_by`)

- [x] The movies should be in chronological order by their `date_watched` which is to say, the first movie chosen should be at the top, with the most current movies at the bottom.

File: src/containers/MovieList/MovieList.scss (hunk 1 above)
Comment (nathanielmillard) [reply src/containers/MovieList/MovieList.scss:10]: Taken care of! Thanks!

**Comment by nathanielmillard** [comment]
It looks like the previous watch data doesn't have the run time stored, so I am going to have some error handling to only include the run time if it's listed until we can updated the data.

**Review by sschipke — APPROVED** [review]

**Pull request merged**